include/d/d_stage.h

```diff
@@ -909,6 +909,15 @@ int dStage_mapInfo_GetOceanZ(stage_map_info_class*);
 int dStage_mapInfo_GetOceanX(stage_map_info_class*);
 void dStage_infoCreate();
 
+struct dStage__ObjectNameTable {
+    char mName[8];
+    s16 mProcName;
+    s8 mSubtype;
+    s8 mGbaName;
+};
```

Comment:
pretty sure i already implemented this so this is a duplicate. one of the structs needs to be removed.